index.html

```diff
@@ -543,6 +543,99 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="production">
+      <h2>Production</h2>
+
+      <p>
+        This section describes how to produce a VC-JWT encoded
+        <code>VerifiableCredential</code> from a <code>Credential</code>.
+      </p>
+
+      <p class="advisment">
+        There are currently 2 competing solutions to this problem. It is a goal
+        of the v2 work to define a breaking change to the specification that
+        reduces production rules to a single, simple, set of instructions that
+        any implementer can easily meet if they posses a software library
+        supporting [[RFC7515]] or [[RFC7519]].
```

Comment:
Assuming this text will be removed later on... but breaking changes themselves are not the goal if they are needed to meet the goal of clarifying production rules, we will introduce them ;)

```suggestion
        There are currently 2 competing solutions to this problem. It is a goal
        of the v2 work to resolve them and
        reduce production rules to a single, simple, set of instructions that
        any implementer can easily meet if they poss a software library
        supporting [[RFC7515]] or [[RFC7519]].
```